config/configurationui.py

```diff
@@ -39,6 +39,11 @@ def __init__(self, project_configuration, master):
         '''
```

Comment:
I did something wrong.... and now your files exist in too places in master. The main location of the file should be "/azure_utils/configurationui.py"

Reply:
Don't worry about that for this commit. Ill dedup after. I can't touch it without my auto formater going crazy and it will just be easier to do after this PR is complete.

Reply:
I saw that, but mentioned earlier I think /config isn't such a bad thing to have to be honest. It's an all in one shop on how to use it...but I don't have a super strong opinion on this.